packages/forge/blocks/perplexity/src/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const perplexityModels = [
+  "sonar-reasoning-pro",
+  "sonar-reasoning",
+  "sonar-pro",
+  "sonar",
+] as const;
+
+export const defaultPerplexityOptions = {
+  model: "sonar",
+  temperature: 1,
+  maxTokens: 1024,
+} as const;
+
+export const API_URL = "https://api.perplexity.ai/chat/completions";
```

Comment:
We don't use all caps constant names throughout the project
```suggestion
export const defaultBaseUrl = "https://api.perplexity.ai";
```